### PR TITLE
Testimonial Temp Fix

### LINKED ISSUE #194: Testimonial Temp Fix
We have had multiple users—internally and externally—struggle to navigate through the desktop testimonials! Typically, users don't know that the cacti are responsible for the navigation of testimonials

### Current State:
Screen Shot 2023-06-19 at 4 28 13 PM

### Your job is to

- [x] Add small numbers to the bottom of the cacti to paginate the testimonials
- [x] Numbers should be black but yellow when selected
- [x] Update the cacti to not be blurry
- [x] Make the selected testimonial a bit more obvious (make !selected cacti state smaller)
- [x] numbers should unmount at mobile view 

### Final State inspo
normal-google-pagination-1565868621

### PR DESCRIPTION
**Pull request by jmsargent10**
Added numbers below testimonial section cacti and resized non-toggled cacti

Fixes #194

Changelist:
- Added numbers to be mapped below each cactus
        - isToggled is determined by is the testimonialIndex is the currentIndex
        - StyledTestimonialNumbers has font styling and color is determined by isToggled from StyledCactusButtonProps
        - added StyledButtonNumberContainer to display each cactus/number pair inline together 
- Lowered width and height for !isToggled cactus buttons to have a larger differentiation in size between selected and non-selected
- Re-exported cacti png files from figma with 4x pixels for crisper resolution

Tested:
- Ran main-site to adjust non-selected cacti sizing and display page numbers in correct location and font styling

Screenshots & Screencasts:
Screen Shot 2023-06-25 at 2 05 52 AM

### CODE REVIEW

**Review by NelsonDong41 — APPROVED** [review]
LGTM!

**Review by mike10911 — CHANGES_REQUESTED** [review]
WOW JOHNY!! IM SO PROUD, YOU KILLED IT. Awesome job integrating with existing state controls! Only a couple changes since this is so clean!

- I see that you removed the `margin-bottom` property in `StyledCactusButtons`. This was originally put there to horizontally align the cactus buttons to ensure that the cacti don't look like they are jumping up when selected—like they are now. Let's put that back!

File: main-site/components/testimonials-section/testimonials/Testimonials.styles.ts [hunk 1]
```diff
@@ -41,19 +43,31 @@ const StyledTestimonialsRightContainer = styled.div`
 
 const StyledCactusButtons = styled.img<StyledCactusButtonsProps>`
   width: ${(StyledCactusButtonsProps): string =>
-    StyledCactusButtonsProps.isToggled ? '4em' : '3em'};
+    StyledCactusButtonsProps.isToggled ? '4em' : '2.7em'};
```
Comment (mike10911) [inline main-site/components/testimonials-section/testimonials/Testimonials.styles.ts]: I think 2.8 works better

**Review by mike10911 — CHANGES_REQUESTED** [review]

**Comment by jmsargent10** [comment]
> WOW JOHNY!! IM SO PROUD, YOU KILLED IT. Awesome job integrating with existing state controls! Only a couple changes since this is so clean!
> 
> * I see that you removed the `margin-bottom` property in `StyledCactusButtons`. This was originally put there to horizontally align the cactus buttons to ensure that the cacti don't look like they are jumping up when selected—like they are now. Let's put that back!

Ahhh yes my bad I see it now, at first I didn't see the effect!

**Review by mike10911 — APPROVED** [review]
LGTM

**Review by kzeng24 — APPROVED** [review]

**Review by Mandy-cyber — APPROVED** [review]
Sorry I'm late, This looks so good!! 10/10 would recommend

**Pull request merged**